### PR TITLE
NUnit update

### PR DESCRIPTION
**Pull request by SignpostMarv**
Had a couple of issues getting the tests to run under nunit-gui, so I've bumped up the nunit version to 3.0.1

### CODE REVIEW

File: VerbalExpressionsUnitTests/SanatizeTests.cs [hunk 1]
```diff
@@ -8,7 +8,6 @@ namespace VerbalExpressionsUnitTests
     public class SanatizeTests
```
Comment (thomasbiel) [inline VerbalExpressionsUnitTests/SanatizeTests.cs]: btw - the file name has a typo

Comment (SignpostMarv) [inline VerbalExpressionsUnitTests/SanatizeTests.cs]: To quote Homer Simpson, "it was like that when I got here".

aka "outside the scope of this pull request".

Comment (thomasbiel) [inline VerbalExpressionsUnitTests/SanatizeTests.cs]: updating nunit also did not involve "running format document on all files"

Comment (SignpostMarv) [inline VerbalExpressionsUnitTests/SanatizeTests.cs]: Given that it required removing the now-unsupported exception attributes, that VS reported line ending issues in some files & git reported whitespace errors, I felt that having a consistent formatting on the files prior to updating nunit was preferable to performing the formatting afterwards.

Comment (thomasbiel) [inline VerbalExpressionsUnitTests/SanatizeTests.cs]: I'm the last one to argue about formatting - obviously the code was dritten without resharper...
Greetings
Am 02.02.2016 19:22 schrieb SignpostMarv <notifications@github.com>:In VerbalExpressionsUnitTests/SanatizeTests.cs:
> @@ -8,7 +8,6 @@ namespace VerbalExpressionsUnitTests
>      public class SanatizeTests

Given that it required removing the now-unsupported exception attributes, that VS reported line ending issues in some files & git reported whitespace errors, I felt that having a consistent formatting on the files prior to updating nunit was preferable to performing the formatting afterwards.

**Comment by psoholt** [comment]
I would love to update to Nunit 3.0. And you have done a great job updating the tests. Still I would like to wait a bit until Nunit 3 is supported in Xamarin Studio too...

**Comment by SignpostMarv** [comment]
> [We may be able to include it in 6.1. I don't have specific dates for this, but given all the time it usually takes to verify and stabilize a release, it won't be sooner than 3 months from now.](https://bugzilla.xamarin.com/show_bug.cgi?id=35915#c12)

Shall we check back on this in aprilish ?

**Comment by psoholt** [comment]
Lets do :)

**Comment by psoholt** [comment]
It seems to work with the new alpha version of Xamarin studio, when I merged the code with the latest commits on master. I did some changes to the project file to make it work as well. Can you check my updated pull request #45  @SignpostMarv  ?

**Pull request closed without merging**